survival_lib.py

```
@@ -16,6 +16,7 @@
 - Vectorized CI extraction (10x faster)
 """
 
+from typing import Union, Optional, List, Dict, Tuple, Any, Sequence
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)